config/routes.rb

```diff
@@ -155,7 +156,4 @@
     end
   end
 
-  # Any other routes are handled here
-  match '*path', to: 'application#routing_error'
```

Comment:
I found that this solution wasn't working in Rails 4 so I switched to using the `exceptions_app` configuration in `config/application.rb`

We used to do it so we could render a 404 page for routes like https://accounts.openstax.org/abc and https://accounts.openstax.org/%EB

But in Rails 4 it was working only for the first and not the second route above.